CHANGELOG.md

```diff
@@ -1,98 +1,89 @@
 # Logic Apps Designer
-## [2.26.0](https://github.com/Azure/LogicAppsUX/compare/v2.25.0...v2.26.0) (2023-06-08)
 
+## [2.26.0](https://github.com/Azure/LogicAppsUX/compare/v2.25.0...v2.26.0) (2023-06-08)
 
 ### Features
 
-* **Data Mapper:** Functions will remain during navigation ([#2665](https://github.com/Azure/LogicAppsUX/issues/2665)) ([d9ef24c](https://github.com/Azure/LogicAppsUX/commit/d9ef24c9e66defae2ee15a72914dbfaef052721c))
-* **Designer:** Browse View - Hiding connectors with no valid actions/triggers ([#2654](https://github.com/Azure/LogicAppsUX/issues/2654)) ([2aa81af](https://github.com/Azure/LogicAppsUX/commit/2aa81afd82b5e14617bdd36a79668c48d87725d4))
-* **designer:** Initial setup work of the chatbot ([#2691](https://github.com/Azure/LogicAppsUX/issues/2691)) ([aaf33c0](https://github.com/Azure/LogicAppsUX/commit/aaf33c0972a61ac173887b72534faa719aafea9a))
-
+- **Data Mapper:** Functions will remain during navigation ([#2665](https://github.com/Azure/LogicAppsUX/issues/2665)) ([d9ef24c](https://github.com/Azure/LogicAppsUX/commit/d9ef24c9e66defae2ee15a72914dbfaef052721c))
```

Comment:
The changelog is getting updated to remove * to -, will this cause any change in how it is viewed in vscode? Can we try to not include this in the PR

Reply:
Yeah I can revert this, this keeps happening automatically in my PRs and I'm really not sure why

Reply:
I think this is a prettier setting, both do the same thing in markdown of creating an unordered list, but I think we should change our release automation to make it so that it uses - instead of * so it aligns with prettier (or we can change or prettier setting)